packages/mobile/src/escrow/saga.ts

```diff
@@ -46,24 +46,31 @@ function* transferStableTokenToEscrow(action: EscrowTransferPaymentAction) {
   Logger.debug(TAG + '@transferToEscrow', 'Begin transfer to escrow')
   try {
     const { phoneHash, amount, tempWalletAddress } = action
-    const escrow: Escrow = yield call(getEscrowContract, web3)
-    const stableToken: StableToken = yield call(getStableTokenContract, web3)
     const account: string = yield call(getConnectedUnlockedAccount)
 
+    const stableToken: StableTokenWrapper = yield call([
+      contractKit.contracts,
+      contractKit.contracts.getStableToken,
+    ])
+    const escrow: EscrowWrapper = yield call([
+      contractKit.contracts,
+      contractKit.contracts.getExchange,
+    ])
+
     Logger.debug(TAG + '@transferToEscrow', 'Approving escrow transfer')
     const convertedAmount = web3.utils.toWei(amount.toString())
-    const approvalTx = stableToken.methods.approve(escrow.options.address, convertedAmount)
+    const approvalTx = stableToken.approve(escrow.address, convertedAmount)
 
-    yield call(sendTransaction, approvalTx, account, TAG, 'approval')
+    yield call(sendTransaction, approvalTx.txo, account, TAG, 'approval')
```

Comment:
This pattern of extracting out the tx object is what I did in verification too. I haven't thought much about a more elegant way to handle this but there probably is one. No need to fix here, just thinking out loud